ios/Sources/TimetableFeature/List/TimetableFeature.swift

```diff
@@ -25,7 +25,17 @@ public enum TimetableStateType {
 public enum TimetableAction {
     case refresh
     case refreshResponse(Result<[AnyTimetableItem], KotlinError>)
-    case loaded(TimetableLoadedAction)
+    case showSetting
+    case none
+
+    init(action: TimetableLoadedAction) {
+        switch action {
+        case .selectedPicker, .content, .detail, .hideDetail:
+            self = .none
+        case .showSetting:
+            self = .showSetting
+        }
+    }
```

Comment:
Is this needed?

it can be implemented with `.loaded(.showSetting)` case in reducer.